hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -688,6 +663,48 @@ public OmKeyInfo lookupKey(OmKeyArgs args, String clientAddress)
     }
   }
 
+  /**
+   * Refresh pipeline info in OM by asking SCM.
+   * @param value OmKeyInfo
+   */
+  @VisibleForTesting
+  protected void refreshPipeline(OmKeyInfo value) {
+    // Refresh container pipeline info from SCM
+    // based on OmKeyArgs.refreshPipeline flag
+    // 1. Client send initial read request OmKeyArgs.refreshPipeline = false
```

Comment:
RpcClient send initial only with refreshPipeline to true


RpcClient.java
Line 661:
    OmKeyArgs keyArgs = new OmKeyArgs.Builder()
        .setVolumeName(volumeName)
        .setBucketName(bucketName)
        .setKeyName(keyName)
        .setRefreshPipeline(true)
        .setSortDatanodesInPipeline(topologyAwareReadEnabled)
        .build();
    OmKeyInfo keyInfo = ozoneManagerClient.lookupKey(keyArgs);

Reply:
@bharatviswa504 I only moved the refresh pipeline logic to a new method since I plan to use it in HDDS-2188. I (hope) have not changed any logic.

Reply:
As in code comments, it is said initially with refreshPipeline, we should update the code comments. My comment is regarding that.

Reply:
You are right @bharatviswa504. Removed the comments that are obsolete now.